cognite/powerops/resync/config/market/dayahead.py

```diff
@@ -39,5 +39,15 @@ def json_loads(cls, value):
         return {"operations": json.loads(value)} if isinstance(value, str) else value
 
     @field_validator("price_scenarios", mode="before")
```

Comment:
I suggest doing this check post as I guess it will not require the `ast.literal_eval`. 

```suggestion
    @field_validator("price_scenarios", mode="after")
```